### PR TITLE
Option to enable/disable individual smartbanner elements

### PR DESCRIPTION
**Pull request by SpaceOgre**
Added individual settings for the information in the smartbanner. See #134 

Options added for:
* Unmanaged citizen icon
* District icons
* Turns to population growth
* Turns to cultural growth

Did some small layout changes to the CityView settings page.
Changed the color for the culture growth number.

Not sure how you usually handle LOCs but did my best to keep new ones to a minimum.
New LOCs:
* LOC_CQUI_CITYVIEW_SMARTBANNER_HEADER
* LOC_CQUI_CITYVIEW_SMARTBANNER_UNLOCKEDCITIZEN_TOOLTIP
* LOC_CQUI_CITYVIEW_SMARTBANNER_DISTRICTS_TOOLTIP
* LOC_CQUI_CITYVIEW_SMARTBANNER_POPULATION_TOOLTIP
* LOC_CQUI_CITYVIEW_SMARTBANNER_CULTURAL_TOOLTIP

### CODE REVIEW

**Comment by chaorace** [comment]
Hey sorry about the delay in getting this reviewed! If all is well expect this to get merged sometime tomorrow morning

**Comment by SpaceOgre** [comment]
Okay, great :) 
No hurry on my end.

File: Assets/CQUI_Settings.sql [hunk 1]
```diff
@@ -30,6 +30,10 @@ INSERT INTO CQUI_Settings -- Don't touch this line!
       ("CQUI_ShowUnitPaths", 1), -- Shows unit paths on hover and selection
       ("CQUI_ShowYieldsOnCityHover", 1), -- Shows city management info like citizens, tile yields, and tile growth on hover
       ("CQUI_Smartbanner", 1), -- Additional informations such as districts will show in the city banner
+	  ("CQUI_Smartbanner_UnlockedCitizen", 1), -- Shows if city have Unmanaged citizens in the banner
```
Comment (chaorace) [inline Assets/CQUI_Settings.sql]: The default indent for CQUI is double-space, as opposed to the tab character. Please correct this section to double-space, thanks! Consult the first bullet in the "General guideline" section of the [Style Guideline](https://github.com/chaorace/cqui/wiki/Style-Guideline) for more information

Comment (SpaceOgre) [inline Assets/CQUI_Settings.sql]: Damn, thought I fix all of those. Will do it later today.

**Pull request merged**